submissions/Harnytskyi/html_css_popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
Unicode is mandatory for symbols in html document, could you please add it?
`<meta charset="UTF-8">`